package.json

```diff
@@ -1,61 +1,29 @@
 {
   "name": "lint-staged",
   "description": "Lint files staged by git",
-  "main": "index.js",
-  "bin": {
-    "lint-staged": "index.js"
-  },
+  "license": "MIT",
+  "repository": "https://github.com/okonet/lint-staged",
+  "author": "Andrey Okonetchnikov <andrey@okonet.ru>",
+  "maintainers": [
+    "Lufty Wiranda <lufty.wiranda@gmail.com",
+    "Suhas Karanth <sudo.suhas@gmail.com"
+  ],
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=4.2.0"
   },
+  "bin": "index.js",
   "files": [
     "src"
   ],
   "scripts": {
+    "pre-commit": "remove-lockfiles && node index.js",
     "commit": "git-cz",
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
-    "pre-commit": "remove-lockfiles && node index.js",
-    "pretest": "npm run lint",
-    "test": "jest --coverage",
-    "deps": "npm-check -s",
-    "deps:update": "npm-check -u"
-  },
-  "pre-commit": "pre-commit",
-  "jest": {
-    "testEnvironment": "node",
-    "setupFiles": [
-      "./testSetup.js"
-    ]
-  },
-  "greenkeeper": {
-    "ignore": [
-      "cosmiconfig"
-    ]
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/okonet/lint-staged.git"
-  },
-  "keywords": [
-    "lint",
-    "git",
-    "staged",
-    "javascript",
-    "css",
-    "scss",
-    "sass",
-    "eslint",
-    "stylelint",
-    "code",
-    "quality"
-  ],
-  "author": "Andrey Okonetchnikov <andrey@okonet.ru>",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/okonet/lint-staged/issues"
+    "test": "jest",
```

Comment:
What's the motivation of going to this instead of having `--coverage` here? Also why we removed `pretest` etc?

Reply:
https://github.com/okonet/lint-staged/pull/250#pullrequestreview-60625243

Reply:
No, the question is what's the motivation to not use `npm test` as a default task on CI. What else do we need `test:coverage` for?